src/lib/addons/feature-event-formatter-md-events.ts

```diff
@@ -0,0 +1,359 @@
+        path: '/admin/groups',
+    },
+    [GROUP_UPDATED]: {
+        label: 'Group updated',
+        action: '{{b}}{{user}}{{b}} updated group {{b}}{{event.preData.name}}{{b}}',
+        path: '/admin/groups',
+    },
+    [BANNER_CREATED]: {
+        label: 'Banner created',
+        action: '{{b}}{{user}}{{b}} created banner {{b}}{{event.data.message}}{{b}}',
+        path: '/admin/message-banners',
+    },
+    [BANNER_DELETED]: {
+        label: 'Banner deleted',
+        action: '{{b}}{{user}}{{b}} deleted banner {{b}}{{event.preData.message}}{{b}}',
+        path: '/admin/message-banners',
+    },
+    [BANNER_UPDATED]: {
+        label: 'Banner updated',
+        action: '{{b}}{{user}}{{b}} updated banner {{b}}{{event.preData.message}}{{b}}',
+        path: '/admin/message-banners',
+    },
+    [PROJECT_CREATED]: {
+        label: 'Project created',
+        action: '{{b}}{{user}}{{b}} created project {{b}}{{project}}{{b}}',
+        path: '/projects',
+    },
+    [PROJECT_DELETED]: {
+        label: 'Project deleted',
+        action: '{{b}}{{user}}{{b}} deleted project {{b}}{{event.project}}{{b}}',
+        path: '/projects',
+    },
+    [SEGMENT_CREATED]: {
+        label: 'Segment created',
+        action: '{{b}}{{user}}{{b}} created segment {{b}}{{event.data.name}}{{b}}',
+        path: '/segments',
+    },
+    [SEGMENT_DELETED]: {
+        label: 'Segment deleted',
+        action: '{{b}}{{user}}{{b}} deleted segment {{b}}{{event.preData.name}}{{b}}',
+        path: '/segments',
+    },
+    [SEGMENT_UPDATED]: {
+        label: 'Segment updated',
+        action: '{{b}}{{user}}{{b}} updated segment {{b}}{{event.preData.name}}{{b}}',
+        path: '/segments',
+    },
+    [SERVICE_ACCOUNT_CREATED]: {
+        label: 'Service account created',
+        action: '{{b}}{{user}}{{b}} created service account {{b}}{{event.data.name}}{{b}}',
+        path: '/admin/service-accounts',
+    },
+    [SERVICE_ACCOUNT_DELETED]: {
+        label: 'Service account deleted',
+        action: '{{b}}{{user}}{{b}} deleted service account {{b}}{{event.preData.name}}{{b}}',
+        path: '/admin/service-accounts',
+    },
+    [SERVICE_ACCOUNT_UPDATED]: {
+        label: 'Service account updated',
+        action: '{{b}}{{user}}{{b}} updated service account {{b}}{{event.preData.name}}{{b}}',
+        path: '/admin/service-accounts',
+    },
+    [USER_CREATED]: {
+        label: 'User created',
+        action: '{{b}}{{user}}{{b}} created user {{b}}{{event.data.name}}{{b}}',
+        path: '/admin/users',
+    },
+    [USER_DELETED]: {
+        label: 'User deleted',
+        action: '{{b}}{{user}}{{b}} deleted user {{b}}{{event.preData.name}}{{b}}',
+        path: '/admin/users',
+    },
+    [USER_UPDATED]: {
+        label: 'User updated',
+        action: '{{b}}{{user}}{{b}} updated user {{b}}{{event.preData.name}}{{b}}',
+        path: '/admin/users',
+    },
+};
```

Comment:
I think it makes sense to split the previous code and put the events formatting map in its own file. It's still pretty big, but hopefully more manageable.